src/main.ts

```diff
@@ -1,35 +1,39 @@
-import * as path from "path";
-import { App, Stack, StackProps, aws_s3 } from "aws-cdk-lib";
+import * as path from "node:path";
+import { fileURLToPath } from "node:url";
+import { App, aws_s3, Stack, type StackProps } from "aws-cdk-lib";
 import { RustFunction } from "cargo-lambda-cdk";
-import { Construct } from "constructs";
+import type { Construct } from "constructs";
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
```

Comment:
You can remove these. I will provide a suggestion for line 20, where this is used.